tech-far-hub/gatsby-browser.tsx

```diff
@@ -0,0 +1,5 @@
+export const onRouteUpdate = ({ location, prevLocation }) => {
+  if (location.pathname === "/contact/" && prevLocation && touchpointFormfbdc7110) {
+    touchpointFormfbdc7110.loadHtml();
```

Comment:
How did you find this and what does this do?

Reply:
Touchpoints loads a script to create the various forms (one script per form), so honestly I just read the script --- it's not obfuscated, thankfully. `loadHtml` replaces my empty div with the form, which is what I needed to happen on every _second_ visit to the page.